app/src/main/java/com/jerboa/api/Http.kt

```diff
@@ -251,18 +253,22 @@ interface API {
 
         fun changeLemmyInstance(instance: String): API {
             currentInstance = instance
-            api = buildApi()
+            api = buildApi(buildUrl())
             return api!!
         }
 
         fun getInstance(): API {
             if (api == null) {
-                api = buildApi()
+                api = buildApi(buildUrl())
             }
             return api!!
         }
 
-        private fun buildApi(): API {
+        fun createTempInstance(host: String): API {
```

Comment:
What is this about?

Reply:
I needed to create a temporary  API Instance using the new host to test if its a Lemmy instance. Currently that was not possible without removing the existing API.